### PR TITLE
adding Ghostery Browser for iOS

### PR DESCRIPTION
**Pull request by ebal**
1. [x] Project URL: https://www.ghostery.com/mobile/
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by dkhamsing** [comment]
@ebal could u add the license field as well pleas 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: contents.json [hunk 1]
```diff
@@ -1254,8 +1254,11 @@
             "title": "Ghostery",
             "category-ids": ["browser"],
             "tags": ["swift"],
+            "homepage": "https://www.ghostery.com/mobile/",
             "description": "Ghostery Browser",
             "source": "https://github.com/ghostery/browser-ios",
+            "license": "Mozilla Public License 2.0",
```
Comment (vpeschenkov) [inline contents.json]: @ebal could you replace it by the following code according to the contribution guidelines `mpl-2.0`

> license: Please use the key name from https://developer.github.com/v3/licenses/ (use other if license info missing)

**Comment by vpeschenkov** [comment]
@ebal thank you 👍

**Comment by ebal** [comment]
thanks for your help

**Pull request merged**